### PR TITLE
BI-10358 Display updated registered office stop screen

### PR DESCRIPTION
**Pull request by mattch1**

### CODE REVIEW

File: views/incorrect-information/wrong-ro.html [hunk 1]
```diff
@@ -9,24 +9,105 @@
...
-        <p><a href="https://www.smartsurvey.co.uk/s/fileconfstmnt-wrongro/">What did you think of this service?</a>
-        (takes 2 minutes)</p>
+        <main class="govuk-main-wrapper govuk-main-wrapper--auto-spacing" id="main-content" role="main">
+
+          <h1 class="govuk-heading-l">
+            You need to update the company details
+          </h1>
+
+          <ol class="app-step-nav__steps">
+            <li class="app-step-nav__step js-step step-is-shown" id="check-youre-allowed-to-drive">
```
Comment (markpit) [inline views/incorrect-information/wrong-ro.html:19]: these id's will have to be checked with UX, looks lie they've come from the dvla and we can't use them in our code. The aria ones will be read by screen readers
 id="check-youre-allowed-to-drive"
aria-controls="step-panel-driving-lessons-and-practice-3"/>
id="get-a-provisional-licence"
etc...

File: views/incorrect-information/wrong-ro.html [hunk 2]
```diff
@@ -1,23 +1,101 @@
...
+                <span class="js-step-title-text">
+                  Update your registered office address
+                </span>
+              </h2>
+            </div>
+            <div>
+              <ol class="app-step-nav__list " data-length="1">
+                <li class="app-step-nav__list-item js-list-item">
+                  <p>You will need to
+                    <a href="/change-data/change-address" target="_blank">update the registered office address (opens in a new tab)</a>
```
Comment (markpit) [inline views/incorrect-information/wrong-ro.html:41]: the href=/change-data/change-address is a link to the prototype so it won't work in our app. It needs to be the same url as before which is passed in from the controller - {{changeRoaUrl}}

Comment (markpit) [inline views/incorrect-information/wrong-ro.html:41]: i'd keep the rel="noopener noreferrer" data-event-id="update-roa-link" values in there as the prototype link won't have the matomo event. The rel="noopener noreferrer" is a security recommendation from Sonar i think.

File: views/incorrect-information/wrong-ro.html [hunk 3]
```diff
@@ -1,23 +1,101 @@
...
+                <span class="js-step-title">
+                  <button class="app-step-nav__button app-step-nav__button--title js-step-title-button"/>
+                </span>
+                <span class="js-step-title-text">
+                  Complete your confirmation statement
+                </span>
+              </h2>
+            </div>
+            <div>
+              <p>Return to the <a href="/confirmation-statement/ro">Registered office address</a> section and confirm that an update has been submitted.</p>
```
Comment (markpit) [inline views/incorrect-information/wrong-ro.html:94]: this href-href="/confirmation-statement/ro" doesn't exist in our app, you'll need to find out what screen they want to return to and test the link in your running docker env

File: views/incorrect-information/wrong-ro.html [hunk 4]
```diff
@@ -36,16 +38,16 @@ <h2 class="app-step-nav__title">
             </div>
             <div>
               <ol class="app-step-nav__list " data-length="1">
-                <li class="app-step-nav__list-item js-list-item">
+                <li class="app-step-nav__list-item js-list-item ">
                   <p>You will need to
-                    <a href="{{changeRoaUrl}}" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">update the registered office address (opens in a new tab)</a>
+                    <a href="/change-data/change-address" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">update the registered office address (opens in a new tab)</a>
```
Comment (mwestacott) [inline views/incorrect-information/wrong-ro.html]: Looks like this has regressed and needs to use the **_changeRoaUrl_** again.

File: views/incorrect-information/wrong-ro.html [hunk 5]
```diff
@@ -1,23 +1,103 @@
...
+                <span class="js-step-title">
+                  <button class="app-step-nav__button app-step-nav__button--title js-step-title-button" aria-expanded="false" aria-controls="step-panel-complete-confirmation-statement"/>
+                </span>
+                <span class="js-step-title-text">
+                  Complete your confirmation statement
+                </span>
+              </h2>
+            </div>
+            <div>
+              <p>Return to the <a href="{{changeRoaUrl}}" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">Registered office address</a> section and confirm that an update has been submitted.</p>
```
Comment (markpit) [inline views/incorrect-information/wrong-ro.html:96]: this link in the prototype takes you to the registered office address screen, this link on this branch crashes.

Comment (mattch1) [reply views/incorrect-information/wrong-ro.html:96]: I have raised a bug to deal with this: https://companieshouse.atlassian.net/browse/NCS-851 it is at the top of the backlog don't worry about this now we have the same problem on main.

[the remark this answers is not in the document]
Comment (mattch1) [reply views/incorrect-information/wrong-ro.html:96]: I'll just have to rmeove this for now then.

Comment (markpit) [reply views/incorrect-information/wrong-ro.html:96]: this is a different link, the prototype takes you to the registered office address screen in the task list for this link

**Comment by mattch1** [comment]
UX have confirmed that the numbered headings should not be links at all, so removed these unnecessary buttons.

File: views/incorrect-information/wrong-ro.html [hunk 6]
```diff
@@ -1,23 +1,94 @@
...
+                  </span>
+                </span>
+                <span class="js-step-title-text">
+                  Return to this service
+                </span>
+              </h2>
+            </div>
+            <div>
+              <ol class="app-step-nav__list " data-length="1">
+                <li class="app-step-nav__list-item js-list-item ">
```
Comment (SVarallo) [inline views/incorrect-information/wrong-ro.html:64]: There's an extra space on the end here

File: views/incorrect-information/wrong-ro.html [hunk 7]
```diff
@@ -1,23 +1,94 @@
...
+                <span class="js-step-title-text">
+                  Update your registered office address
+                </span>
+              </h2>
+            </div>
+            <div>
+              <ol class="app-step-nav__list " data-length="1">
+                <li class="app-step-nav__list-item js-list-item ">
+                  <p>You will need to
+                    <a href="{{changeRoaUrl}}" target="_blank" rel="noopener noreferrer" data-event-id="update-roa-link">update the registered office address (opens in a new tab)</a>
```
Comment (SVarallo) [inline views/incorrect-information/wrong-ro.html:40]: Usually we do spaces surrounding the variables like {{ changeRoaUrl }}

**Review by SVarallo — APPROVED** [review]

**Review by markpit — APPROVED** [review]

**Pull request merged**